File: modul 6/PRAK604-2410817320004-AmandhaCitraMustika.py
pesan = input("")
kode = input("")
print(" ")
karakter = ""
bintang = 0
pagar = 0
if len(pesan) == len(kode):
    for i in range (len(pesan)) :
        if pesan[i] == ' ' and kode[i] == ' ' :
            karakter += " "
        elif pesan[i] == kode[i] :
            karakter +="*"
            bintang += 1
        else :
            karakter +="#"
            pagar += 1
    print(karakter)
    print(f"* = {bintang}")
    print(f"# = {pagar}")
    if bintang >= pagar :
        print("Pesan Asli")
    else :
        print("Pesan Palsu")
else :
    print("Panjang kalimat berbeda, pesan palsu")

File: modul 6/PRAK604-2410817320004-AmandhaCitraMustika.c
#include <stdio.h>
#include <string.h>
int main(){
  char kode[100], pesan[100];
  scanf("%[^\n]%*c %[^\n]%*c", &pesan, &kode);
  printf("\n");
  int bintang = 0, pagar = 0, i;
  if(strlen(pesan) == strlen(kode)){
    for(i = 0; i < strlen(pesan); i++){
      if(pesan[i] == ' ' && kode[i] == ' '){
        printf(" ");}
      else if(pesan [i] == kode[i]){
        printf("*");
        bintang++;}
      else if(pesan [i] != kode[i]){
        printf("#");
        pagar++;}
      else{}}
    printf("\n* = %d \n", bintang);
    printf("# = %d \n", pagar);
    if(bintang >= pagar){
      printf("Pesan Asli\n");}
    else{
      printf("Pesan Palsu\n");}}
  else{
    printf("Panjang kalimat berbeda, pesan palsu");}
  return 0;}